include/wx/event.h

```diff
@@ -2919,6 +2921,10 @@ class WXDLLIMPEXP_CORE wxUpdateUIEvent : public wxCommandEvent
     void Show(bool show) { m_shown = show; m_setShown = true; }
     void SetText(const wxString& text) { m_text = text; m_setText = true; }
 
+    // Set a flag saying if the object being updated supports the check action
+    bool IsCheckable() const { return m_isCheckable; }
+    void SetIsCheckable(bool aCheckable) { m_isCheckable = aCheckable; }
```

Comment:
We don't use `aFoo` convention anywhere else, so I'd rather do
```suggestion
    void SetIsCheckable(bool checkable) { m_isCheckable = checkable; }
```